content/en/network_monitoring/netflow/_index.md

```diff
@@ -79,6 +81,18 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
 
 {{< img src="network_device_monitoring/netflow/new_enrichment.png" alt="The New Enrichment Mapping modal in the Netflow configuration tab" width="80%" >}}
 
+### Reverse DNS private IP enrichment
+
+Enable Reverse DNS private IP enrichment to perform DNS lookups to find the hostname name associated with source or destination IP addresses. When enabled, the Agent conducts reverse DNS lookups on source and/or destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.
+
+Reference the [configuration](#configuration) file above to enable Reverse DNS IP enrichment in your `datadog.yaml` file (disabled by [default][7]):
+
+Search for **DNS** in the Flow grouping of the facets section to locate flows associated with reverse DNS IP enrichment: 
```

Comment:
Is it reverse DNS IP enrichment or Reverse DNS IP enrichment? It's capitalized in other places, but not here.